sklearn/pipeline.py

```diff
@@ -69,6 +69,11 @@ class Pipeline(_BaseComposition):
     to another estimator, or a transformer removed by setting it to
     `'passthrough'` or `None`.
 
+    For examples use cases of `Pipeline` combined with
```

Comment:
```suggestion
    For use case examples of :class:`Pipeline` combined with
```

Reply:
Also here, the link would link to the currently open document.